index.bs

```diff
@@ -2549,21 +2565,39 @@ This attestation statement format is used with FIDO U2F authenticators using the
         The attestation certificate must be the first element in the array.
 
     : sig
-    :: The [=attestation signature=].
+    :: The [=attestation signature=].  It contains the (raw) U2F registration response message [[!FIDO-U2F-Message-Formats]].
```

Comment:
s/It contains/This signature was calculated over/

s/message [[!FIDO-U2F-Message-Formats]]/message [[!FIDO-U2F-Message-Formats]] received by the platform from the authenticator/

Reply:
Done.